pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/models/consumer/objects_api/membership/PNChannelMembership.java

```diff
@@ -20,31 +20,80 @@ public static class ChannelId {
 
     @Getter
     private final ChannelId channel;
+    @Getter
+    private final Map<String, Object> custom;
+    @Getter
+    private final String status;
+    @Getter
+    private final String type;
 
+    @Deprecated
```

Comment:
do we need to deprecate this helper? might be still useful for someone who doesn't care about status/type

Reply:
Generally I like removing legacy code if it simplifies our codebase, but in this case I don't see much gain for us in terms of maintenance (we're not switching to some entirely different implementation, it's just a helper), but for the user it might be annoying to see deprecations and have to move to something new. 

If we can keep it because it doesn't break anything for us, then let's also keep it (undeprecated) for the user? 

Unless we really want to remove it at some point, but I don't see that we have to?

Reply:
Ok, I will remove deprecation